packages/unity-bootstrap-theme/stories/docs/content-sections/content-image-overlap.stories.mdx

```diff
@@ -1,6 +1,6 @@
 import { Meta, Story, Canvas } from '@storybook/addon-docs';
 
-<Meta title="Docs/Content Sections/Content Image Overlap" />
+<Meta title="Organisms/Content Sections/Readme" />
```

Comment:
This bit of documentation belongs under `Molecules/ Content Sections/Image Overlap/Readme` instead of here.

Reply:
Also, agreed with everything @scott-williams-az said.